Windows Scripts Update

Zackhardtoname opened the pull request:
1. Added functionality to change body parts and num2frames
2. Refactored bash scripts
3. Updated readme.md
4. gitignoring all DS_Store files

Zackhardtoname committed:
1. Added functionality to change body parts and num2frames
2. Refactored bash scripts
3. Updated readme.md
4. gitignoring all DS_Store files

Zackhardtoname committed:
1. added support for Mac
2. updated readme.md to help solve a potential Conda issue on Windows

shreyapandit reviewed (changes requested):
Minor changes needed to README, otherwise looks simple and elegant

Zackhardtoname committed:
1. Fixed a bug that does not write config.json to the current place
2. README.md has become more clear
3. Created the "resources" folder for learning resources.

Zackhardtoname committed:
moved files around

Zackhardtoname commented:
I have made the requested changes in the recent commits.

Zackhardtoname committed:
added more details in README.md

shreyapandit reviewed (approved):
Looks good!
